config.json

```diff
@@ -46,7 +46,8 @@
     "rectangles",
     "forth",
     "circular-buffer",
-    "react"
+    "react",
+    "atbash-cipher"
```

Comment:
We'll want to move this earlier in the problem track. Around word-count, I think.